openedx/core/djangoapps/content_libraries/serializers.py

```diff
@@ -115,6 +115,7 @@ class BaseFilterSerializer(serializers.Serializer):
     """
     text_search = serializers.CharField(default=None, required=False)
     org = serializers.CharField(default=None, required=False)
+    order = serializers.CharField(default=None, required=False)
```

Comment:
I would prefer a default here (in the serializer/rest api level) so we always return a sorted list, but is up to you.

```suggestion
    order = serializers.CharField(default='title', required=False)
```